Vector/ViewController/SegmentedViewController.m

```diff
@@ -143,6 +143,18 @@ - (void)viewDidLoad
     [self createSegmentedViews];
 }
 
+- (void)viewWillAppear:(BOOL)animated
+{
+    [super viewWillAppear:animated];
+
+    if (_selectedViewController)
+    {
+        // Make iOS invoke child viewWillAppear and viewDidAppear
+        [_selectedViewController beginAppearanceTransition:YES animated:animated];
+        [_selectedViewController endAppearanceTransition];
```

Comment:
That does not work like that.
This is beginAppearanceTransition that make iOS post calls on viewWillAppear and viewDidAppear.
endAppearanceTransition has no direct effect on the call of  viewDidAppear